subsys/bluetooth/host/conn.c

```diff
@@ -214,6 +214,16 @@ static int send_conn_le_param_update(struct bt_conn *conn,
 	       conn->le.features[0], param->interval_min,
 	       param->interval_max, param->latency, param->timeout);
 
+	/* Proceed only if connection parameters contains valid values*/
+	if (!((param->interval_min >= 6 && param->interval_min <= 3200 &&
```

Comment:
Should use `bt_le_conn_params_valid` instead.

Reply:
Already there? Cool ! Done :)